frontend/src/Components/NPC/NPC.css

```diff
@@ -0,0 +1,9 @@
+.NPC {
+  position: absolute;
+  z-index: 1;
+  transition: all 0.2s;
+}
+
+.NPCdead {
```

Comment:
I may be wrong, but I think this class is useless: if your NPC is dead, it's not included in the render at all